docs/ops/matrix/Einsum_7.md

```diff
@@ -0,0 +1,228 @@
+     [[2.0, 4.0, 6.0],
+      [8.0, 10.0, 12.0],
+      [14.0, 16.0, 18.0]]]
+equation = "kii->ki"
+output = [[1.0, 5.0, 9.0],
+          [2.0, 10.0, 18.0]]
+```
+
+Example 5 shows how *Einsum* transposes input tensor:
+
+```
+A = [[[1.0, 2.0, 3.0],
+      [4.0, 5.0, 6.0],
+      [7.0, 8.0, 9.0]]]
+equation = "ijk->kij"
+output = [[[1.0, 4.0, 7.0]],
+          [[2.0, 5.0, 8.0]],
+          [[3.0, 6.0, 9.0]]]
+```
+
+In addition to a lowercase label, ellipsis `...` can be used as a label in a subscript to cover broadcasted dimensions.
+Each input subscript can contain at most one ellipsis. For example, the ellipsis in input subscript `a...bc` for five rank tensor covers 
+the second and third dimensions. In case input subscripts contain ellipsis for several operands, the dimensions covered by the ellipsis
+must be broadcastable to satisfy numpy broadcasting (or multidirectional broadcasting) rules available in
+[Broadcast Rules For Elementwise Operations](../broadcast_rules.md).
+If at least one input subscript contains an ellipsis, the output subscript must always contain one ellipsis.
+For example, *Einsum* operation on two inputs of shapes `[11, 1, 4, 3]` and `[3, 11, 7, 1]` with `equation="a...b,b...->a..."`
+has ellipsis for both operands covering dimensions with sizes `[1, 4]` and `[11, 7, 1]` that are broadcasted to `[11, 7, 4]`.
+The resulted shape of *Einsum* operation will be `[11, 11, 7, 4]` since the dimension labeled with `a` is left with broadcasted dimensions.
+
+Example 6 shows how *Einsum* operates on the single input with an equation containing ellipsis:
+
+```
+A = [[1.0, 2.0, 3.0],
+     [4.0, 5.0, 6.0],
+     [7.0, 8.0, 9.0]]
+equation = "a...->..."
+output = [12.0, 15.0, 17.0]
+```
+
+Example 7 shows how *Einsum* operates with broadcasting two operands:
+
+```
+A = [[1.0, 2.0, 3.0],
+     [4.0, 5.0, 6.0],
+     [7.0, 8.0, 9.0]]
+B = [0.5]
+equation = "a...,...->a..."
+output = [[0.5, 1.0, 1.5],
+          [2.0, 2.5, 3.0],
+          [3.5, 4.0, 4.5]]
+```
+
+**NOTE**: Deep learning frameworks support implicit mode (a classical form of Einstein summation) where the equation does not have the output subscript 
+and has the following format: `<subscript for input1>, <subscript for input2>, ..., <subscript for inputn>`.
+The equation consists of only input subscripts for each operand.
+The output subscript can be recovered as a sequence of alphabetically sorted labels that are not repeated in the left-hand side of the equation.
+For example, `equation = "dbbc,ca"` in implicit mode is the same as `equation = "dbbc,ca->ad"`.
+The equation in implicit mode can set up only subset of Einstein summation conventions, so this mode has limitations. For example,
+`equation = "kii->i"` cannot be represented in implicit mode. *Einsum* operation only supports explicit mode.
+In case ellipsis label is in the left-hand side of the equation in implicit mode, the ellipsis comes first in the output subscript for the recovery.
```

Comment:
Why do we limit our customers to only explicit mode? We can easily convert the implicit mode to explicit mode internally for easier handling by plugin, but from the API perspective I vote for allowing implicit mode.

Reply:
Implicit mode can be easily represented with explicit mode but not vise versa. We can offload recovering output subscript into importer, MO and different other bridge.

Reply:
What if customer builds the model using C++ or Python API? In this case there is no any "importer" in between and they are forced to use explicit mode.

Reply:
In any case the customer has to pay attention to that our opset is different from frameworks opset. I think it is a drawback not to have the output subscript that I am going to fix in our opset because the implicit mode for all frameworks does not declare definite order of dimensions.

Reply:
We agreed to support implicit mode and include capital letters in equation. The text is changed based on this comment.